docs/source/features/quantization/index.md

```diff
@@ -12,6 +12,7 @@ supported_hardware
 auto_awq
 bnb
 gguf
+gptqmodel
```

Comment:
This list appears to be doing a-z order but `fp8` is near the bottom. I would actually re-order this in the performance and level of vllm kernel support, not pkg name. bnb and gguf should be near the bottom here but not my decision to make.